torch/ao/quantization/quantizer/xnnpack_quantizer_utils.py

```diff
@@ -0,0 +1,506 @@
+import itertools
+import operator
+from typing import Callable, Dict, List, Optional
+
+import torch
+from torch.ao.quantization.pt2e.graph_utils import find_sequential_partitions
+from torch.ao.quantization.quantizer import QuantizationSpecBase, SharedQuantizationSpec
+
+# TODO: move these to this file
```

Comment:
should we still have this TODO? Those seem like pretty general quantizer util functions

Reply:
we do we want to move some of them to this file, see other PRs in the stack